jsonschema-core.md

```diff
@@ -1386,28 +1381,13 @@ applicator keywords or with location-reserving keywords such as
 be `$defs` and the standard applicators from this document or
 implementation-specific custom keywords.
 
-Multi-level structures of unknown keywords are capable of introducing nested
-subschemas, which would be subject to the processing rules for `$id`. Therefore,
-having a reference target in such an unrecognized structure cannot be reliably
-implemented, and the resulting behavior is undefined. Similarly, a reference
-target under a known keyword, for which the value is known not to be a schema,
-results in undefined behavior in order to avoid burdening implementations with
-the need to detect such targets.[^10]
```

Comment:
This seems to me to be a useful thing to address a longstanding issue/ambiguity , why remove it? Addressing schemas in unknown locations should be called out as undefined behavior.

Reply:
Unknown keywords are no longer a thing, so calling out data structures under them no longer makes sense.

Reply:
Oh right, I'd forgotten that part, that's fine then.

Reply:
I'm unresolving this conversation because my comment wasn't addressed. I assume it was missed. The second part of this paragraph is still relevant. It deals with known keywords. That means the footnote is still relevant as well, but we don't necessarily need to keep it.

> Similarly, a reference target under a _known keyword_, for which the value is known not to be a schema, results in undefined behavior in order to avoid burdening implementations with the need to detect such targets.[^10]

Reply:
Yeah I missed it. I think there was a concurrency error between me resolving the thread and you adding your comment.

I'll add that bit about known keywords back in.